operations/app/src/modules/database/main.tf

```diff
@@ -18,6 +18,11 @@ resource "azurerm_postgresql_server" "postgres_server" {
   ssl_enforcement_enabled = true
   ssl_minimal_tls_version_enforced = "TLS1_2"
 
+  threat_detection_policy {
+    enabled = true
+    email_account_admins = true
```

Comment:
This configuration will send alerts to `Azure Security Center`.  Based on terraform's docs, it can also be configured to be sent to a storage account if we need to do that instead.

Reply:
This looks good. Emailing is great start, and we can integrate with Splunk if there is a desire from that team.